README.md

```diff
@@ -379,6 +379,51 @@ func main() {
     // ...
 }
 ```
+
+As OpenTelemetry has now reached feature parity with OpenCensus, the migration
+from OpenCensus to OpenTelemetry is strongly encouraged.
+[OpenTelemetry bridge](https://github.com/open-telemetry/opentelemetry-go/tree/main/bridge/opencensus)
+can be leveraged to migrate to OpenTelemetry without the need of replacing the
+OpenCensus APIs in this library. An example code is shown below for application
+to initialize the exporter using the OpenTelemetry bridge for traces.
+
+```golang
+import (
+	texporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
+	"go.opencensus.io/trace"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/bridge/opencensus"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"google.golang.org/api/option"
+)
+
+func main() {
+	// Be careful about using trace.AlwaysSample in a production application
+	// with significant traffic: a new trace will be started and exported for
```

Comment:
nit: I find this comment block a bit confusing...

Reply:
// trace.AlwaysSample() is expensive. Recommend replacing it with your own 
    // sampler for your production.
How does this look?

Reply:
maybe a slight tweak?

```
// trace.AlwaysSample() is expensive. Replacing it with your own 
// sampler for production environments is recommended.
```